Epilepsy/epilepsy_utils.py

```diff
@@ -0,0 +1,119 @@
+from collections import defaultdict
+from itertools import product
+from string import punctuation
+
+from pandas import DataFrame
+
+def offsets_to_token(left, right, offset_array, lemmas, punc=set(punctuation)):
```

Comment:
Even though it doesn't matter in every case, the habit of setting equal to `None` and then testing for equality and setting within the function will save you a day of debugging at least once in your life. Unless performance reasons preclude it, I'd do that every time to save the day  :+1: